src/Components/test/E2ETestMigration/Microsoft.AspNetCore.Components.Migration.E2ETests.csproj

```diff
@@ -24,13 +24,21 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\E2ETest\Microsoft.AspNetCore.Components.E2ETests.csproj" ReferenceOutputAssembly="false" SkipGetTargetFrameworkProperties="true" />
     <ProjectReference Include="$(RepoRoot)src\Shared\BrowserTesting\src\Microsoft.AspNetCore.BrowserTesting.csproj" />
     <ProjectReference Include="..\..\WebAssembly\testassets\HostedInAspNet.Client\HostedInAspNet.Client.csproj" />
     <ProjectReference Include="..\..\WebAssembly\testassets\HostedInAspNet.Server\HostedInAspNet.Server.csproj" />
     <ProjectReference Include="..\..\WebAssembly\testassets\StandaloneApp\StandaloneApp.csproj" />
     <ProjectReference Include="..\testassets\BasicTestApp\BasicTestApp.csproj" />
     <ProjectReference Include="..\testassets\GlobalizationWasmApp\GlobalizationWasmApp.csproj" />
     <ProjectReference Include="..\testassets\TestServer\Components.TestServer.csproj" />
+
+    <ProjectReference
+      Include="..\testassets\BasicTestApp\BasicTestApp.csproj"
+      Targets="Publish"
+      Properties="PublishDir=$(OutputPath)helixcontent\BasicTestApp\"
```

Comment:
This conflicts with line 32 above; BasicTestApp.csproj will now be built twice with different global properties (once with nothing special, and once with this PublishDir) in parallel, which is a race condition.

Reply:
Thanks, that makes perfect sense!  I'll try removing the top line, the main thing I was trying to accomplish was publish the app so we can copy the output into the helix payload.

Reply:
So I copied this pattern from the other test project here (except it has a TestTrimmedApps = true condition): https://github.com/dotnet/aspnetcore/blob/main/src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj#L58

Do we also have a race condition there?

Reply:
Well this is a problem, we get compile/build errors if I remove the non publish target reference as there are references to types in the basic test app.  I'm not sure what the right fix is now?

Reply:
Perhaps there a better way for me to force the BasicTestApp to publish than the ProjectReference, I just need it to happen before the PublishAssets target that copies the published output